lib/msal-browser/src/request/SilentRequest.ts

```diff
@@ -26,4 +26,5 @@ export type SilentRequest = Omit<CommonSilentFlowRequest, "authority"|"correlati
     account?: AccountInfo;
     correlationId?: string;
     forceRefresh?: boolean;
+    silentTokenRetrievalStrategy?: String;
```

Comment:
You can use the enum as the type, just make sure you import the enum in this file.

```suggestion
    silentTokenRetrievalStrategy?: SilentTokenRetrievalStrategy;
```

This way the type checker can make sure devs only pass supported types in and we have an extra bug detection layer.